packages/theme/tsconfig.json

```diff
@@ -0,0 +1,7 @@
+{
+  "extends": "../../tsconfig.json",
+  "include": ["src"],
+  "compilerOptions": {
+    "noImplicitAny": false
```

Comment:
You're tearing me apart lisa!

Reply:
Ouch, very unintentional copy-pasta! Kill it with fire.